tinyagentos/tools/todo_tools.py

```
@@ -0,0 +1,142 @@
+        return {"error": "todo_list_lists requires an 'owner_user_id' string"}
+
+    try:
+        store = request.app.state.todo_store
+        lists = await store.list_lists(owner_user_id)
+        # Strip internal fields the agent does not need.
+        slim = [
+            {k: v for k, v in doc.items() if k in ("id", "title", "updated_at")}
+            for doc in lists
+        ]
+        return {"lists": slim}
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)